packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -301,7 +308,23 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
 
   if (buildOptions.progress) {
     const ProgressPlugin = require('webpack/lib/ProgressPlugin');
-    extraPlugins.push(new ProgressPlugin({ profile: buildOptions.verbose }));
+    const spinner = new Spinner();
+
+    extraPlugins.push(new ProgressPlugin({
+      handler: (percentage: number, message: string) => {
+        switch (percentage) {
+          case 0:
+            spinner.start(`Generating ${platform} application bundles...`);
+            break;
+          case 1:
```

Comment:
Is the percentage always between 0 and 1?

Reply:
Yes